packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -1575,8 +1575,14 @@ export const studioLocaleStrings = defineLocalesResources('studio', {
   'timeline.error.unable-to-load-revision': 'Unable to load revision',
   /** Label for when the timeline item is the latest in the history */
   'timeline.latest': 'Latest',
-  /** Label for latest version for timeline menu dropdown */
-  'timeline.latest-version': 'Latest version',
+
+  /** Label for latest revision for timeline menu dropdown */
+  'timeline.latest-revision': 'Latest revision',
```

Comment:
@bjoerge I don't see anywhere in the code `timeline.latest-revision` key being used. Is that expected?